packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -92,41 +93,41 @@ export class VaAccordionItem {
   }
 
   render() {
-    // IE11 is unable to run onSlotChange
-    // Check needed to populate Accordion Item Header information
-    // When using slot="headline" to set the information
-    const ieSlotCheckTag = this.el
-      .querySelector('[slot="headline"]')
-      ?.tagName.toLowerCase();
-    const ieSlotCheckHeader =
-      this.el.querySelector('[slot="headline"]')?.innerHTML;
 
-    const Header = () =>
-      h(
-        // TODO: Remove level & header prop & let the slot do its work
-        /* eslint-disable-next-line i18next/no-literal-string */
-        this.slotTag || ieSlotCheckTag || `h${this.level}`,
-        null,
-        <button
-          ref={el => {
-            this.expandButton = el;
-          }}
-          onClick={this.toggleOpen.bind(this)}
-          aria-expanded={this.open ? 'true' : 'false'}
-          aria-controls="content"
-        >
-          {this.slotHeader || ieSlotCheckHeader || this.header}
-          {this.subheader ? <p>{this.subheader}</p> : false}
-        </button>,
+    const Header = () => {
+      const headline = this.el.querySelector('[slot="headline"]');
+      const ieSlotCheckHeader = headline?.innerHTML;
+      const Tag = (headline && headline.tagName.includes('H'))
+        ? headline.tagName.toLowerCase()
+        : `h${this.level}`;
+      console.log(this.bold)
+      const boldness = this.bold ? 'bold' : 'normal';
+      return (
+        <Tag>
+          <button
+            ref={el => {
+              this.expandButton = el;
+            }}
+            onClick={this.toggleOpen.bind(this)}
+            aria-expanded={this.open ? 'true' : 'false'}
+            aria-controls="content"
+            style={{ fontWeight: boldness }}
+          >
+            {< slot name="icon" />}
```

Comment:
Just because it's a named slot doesn't mean it's limited to just icons unless we programmatically only allow icon nodes to be passed into it but if @humancompanion-usds is fine with this approach, I don't have any issues with it. I think it makes sense to have it a slot since we're unsure of which icon library would be used here in the future.

Minor style question: Is it necessary to have `{ }` around the slot?

Reply:
I defer to you both. Seems fine to me.

Reply:
Minor style answer: Those can be removed, It was a ternary thing there before, but that turned out not to be needed.